iris-mpc/src/services/processors/batch.rs

```diff
@@ -381,6 +381,8 @@ impl<'a> BatchProcessor<'a> {
         }
 
         self.delete_message(sqs_message).await?;
+        // skip_persistence is only used for uniqueness requests
+        self.batch_query.skip_persistence.push(false);
```

Comment:
Deletions are not related to this field `skip_persistence` and it should not push a value into it.

The structure `BatchQuery` has multiple independent "tables" inside:
- Uniqueness, reauth, and reset check, with columns: `request_ids`, `request_types`, `metadata`, `or_rule_indices`, and `skip_persistence`.
- Deletions with columns: `deletion_requests_*`.
- Reset updates with columns: `reset_update_*`.